File: Pattern30.py
# E D C B A
# D C B A
# C B A
# B A
# A

n = int(input("Enter the number:"))
val = 64+n
for i in range(1,n+1):
    printable = val
    for j in range(i,n+1):
        print(chr(printable)," ",end='')
        printable = printable - 1 
    val = val -1 
    print()

File: PATTERNS/Pattern30/Pattern30.java
import java.util.Scanner;

// E D C B A
// D C B A
// C B A
// B A
// A

public class Pattern30{
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n;
        System.out.print("Enter the number:");
        n = in.nextInt();
        int val = 64+n;
        for(int i=1;i<=n;i++){
            int printable = val;
            for(int j=i;j<=n;j++){
                System.out.printf("%c ",(char)(printable--));
            }
            val--;
            System.out.println();
        }
    }
}